packages/gatsby-plugin-emotion/src/gatsby-browser.js

```diff
@@ -1,11 +1,3 @@
-/* globals window */
-import { hydrate } from "emotion"
+const { wrapElement } = require(`./wrap-element`)
 
-exports.onClientEntry = () => {
```

Comment:
Just about ready to merge this. Is this no longer required? As best as I could tell, the example works, but not sure if this was previously doing something important!

Reply:
v10 changed how critical style extraction works and I _believe_ we don't have to do anything here. @mitchellhamilton can confirm, but it seems to work.

Reply:
Yes, this is correct.

Reply:
Appreciate you both for chiming in :) Thank you!

@wKovacs64 lets make that ESM change, and I'll get this merged in. It'll be released as `gatsby-plugin-emotion@3.0.0`